{
 "bad": [
  "WindowsError",
  "ctypes.windll.gdi32.GetPixel",
  "ctypes.windll.user32.GetDC",
  "ctypes.windll.user32.ReleaseDC",
  "cv2",
  "cv2.COLOR_BGR2GRAY",
  "cv2.CV_LOAD_IMAGE_COLOR",
  "cv2.CV_LOAD_IMAGE_GRAYSCALE",
  "cv2.IMREAD_COLOR",
  "cv2.IMREAD_GRAYSCALE",
  "cv2.TM_CCOEFF_NORMED",
  "cv2.__version__",
  "cv2.cvtColor",
  "cv2.imread",
  "cv2.matchTemplate",
  "errno",
  "errno.ENOENT",
  "firstFindFunc",
  "locateAll",
  "os.unlink",
  "pyscreeze.__init__.Box",
  "pyscreeze.__init__.GRAYSCALE_DEFAULT",
  "pyscreeze.__init__.ImageNotFoundException",
  "pyscreeze.__init__.LOAD_COLOR",
  "pyscreeze.__init__.LOAD_GRAYSCALE",
  "pyscreeze.__init__.Point",
  "pyscreeze.__init__.RGB",
  "pyscreeze.__init__.RUNNING_CV_2",
  "pyscreeze.__init__.RUNNING_PYTHON_2",
  "pyscreeze.__init__.USE_IMAGE_NOT_FOUND_EXCEPTION",
  "pyscreeze.__init__.__win32_openDC",
  "pyscreeze.__init__._load_cv2",
  "pyscreeze.__init__._locateAll_opencv",
  "pyscreeze.__init__._locateAll_python",
  "pyscreeze.__init__._screenshot_linux",
  "pyscreeze.__init__._screenshot_osx",
  "pyscreeze.__init__._screenshot_win32",
  "pyscreeze.__init__.center",
  "pyscreeze.__init__.locate",
  "pyscreeze.__init__.locateAll",
  "pyscreeze.__init__.locateOnScreen",
  "pyscreeze.__init__.pixel",
  "pyscreeze.__init__.screenshot",
  "pyscreeze.__init__.scrotExists",
  "pyscreeze.__init__.useOpenCV",
  "pyscreeze.__init__.whichProc",
  "screenshot",
  "sys",
  "sys.platform",
  "sys.platform.startswith",
  "sys.version_info",
  "time",
  "time.time"
 ],
 "deps": {
  "PIL": [
   "pillow/conda-forge/linux-64/pillow-4.0.0-py36_1",
   "pillow/conda-forge/linux-64/pillow-4.2.1-py35_1",
   "pillow/conda-forge/linux-64/pillow-4.3.0-py27_2",
   "pillow/conda-forge/linux-64/pillow-4.3.0-py36_1",
   "pillow/conda-forge/linux-64/pillow-7.2.0-py36hfc7c323_2",
   "pillow/conda-forge/linux-64/pillow-8.3.0-py37hf0e1dd1_0"
  ],
  "WindowsError": [],
  "collections": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004",
   "python/conda-forge/linux-64/python-3.7.10-hffdb5ce_100_cpython"
  ],
  "contextlib": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004",
   "python/conda-forge/linux-64/python-3.7.10-hffdb5ce_100_cpython"
  ],
  "ctypes": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004",
   "python/conda-forge/linux-64/python-3.7.10-hffdb5ce_100_cpython"
  ],
  "cv2": [],
  "datetime": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004",
   "python/conda-forge/linux-64/python-3.7.10-hffdb5ce_100_cpython"
  ],
  "errno": [],
  "firstFindFunc": [],
  "locateAll": [],
  "numpy": [
   "numpy/conda-forge/linux-64/numpy-1.13.0-py36_blas_openblas_200",
   "numpy/conda-forge/linux-64/numpy-1.16.5-py38h18fd61f_1",
   "numpy/conda-forge/linux-64/numpy-1.18.5-py36h7314795_0",
   "numpy/conda-forge/linux-64/numpy-1.21.0-py37h038b26d_0",
   "numpy/conda-forge/linux-64/numpy-1.21.1-py37h038b26d_0",
   "numpy/conda-forge/linux-64/numpy-1.9.3-py27_blas_openblashb06ca3d_7"
  ],
  "os": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004",
   "python/conda-forge/linux-64/python-3.7.10-hffdb5ce_100_cpython"
  ],
  "pyscreeze": [],
  "screenshot": [],
  "subprocess": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_2_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004",
   "python/conda-forge/linux-64/python-3.7.10-hffdb5ce_100_cpython"
  ],
  "sys": [],
  "time": []
 }
}